test/settings.py

```diff
@@ -1,11 +1,28 @@
 """Constants used in test files."""
 
-HOST = "127.0.0.1"
-PORT = "5050"
-L1_HOST = "localhost"
-L1_PORT = "8545"
-
-APP_URL = f"http://{HOST}:{PORT}"
-GATEWAY_URL = APP_URL
-FEEDER_GATEWAY_URL = APP_URL
-L1_URL = f"http://{L1_HOST}:{L1_PORT}/"
+import socket
+
+# pylint: disable=invalid-name,too-few-public-methods
+class Settings:
+    '''settings'''
+    HOST = "127.0.0.1"
+    PORT = ""
+    APP_URL = ""
+
+    L1_HOST = "localhost"
+    L1_PORT = ""
+    L1_URL = ""
+
+    def __init__(self):
+        '''Setup app endpoints for testing'''
+        sock = socket.socket()
```

Comment:
Can we extract lines 18-20 and 23-25 to a function called `get_free_port` or `bind_free_port`?